pyvista/plotting/mapper.py

```diff
@@ -1018,9 +1023,9 @@ def use_circular_splat(self, opacity: float = 1.0):
         # maintain consistency with the default style
         self.scale_factor *= 1.5
 
-    def use_default_splat(self):
+    def use_default_splat(self) -> None:
         """Clear the fragment shader and use the default splat."""
-        self.SetSplatShaderCode(None)
+        self.SetSplatShaderCode(None)  # type: ignore[arg-type]
```

Comment:
This ignore is unrelated to `None` annotations. It's because `_BaseMapper` doesn't inherit any vtk methods